src/modules/tccpp/components/skill-role-suggestion.ts

```diff
@@ -19,10 +19,6 @@ import { build_description, capitalize } from "../../../utils/strings.js";
 import { EarlyReplyMode, TextBasedCommandBuilder } from "../../../command-abstractions/text-based-command-builder.js";
 import { TextBasedCommand } from "../../../command-abstractions/text-based-command.js";
 
-type interaction_context = { member: Discord.GuildMember; role?: string; context?: Discord.Message };
-
-type skill_level = "beginner" | "intermediate" | "proficient" | "advanced" | "expert";
```

Comment:
`skill_level` now is going to be uppercase versions of these strings, so this is schema breaking